maintenance/datastore-migration.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate Calico Data from an etcdv3 Datastore to a Kubernetes Datastore
+description: Migrate your cluster from using an etcdv3 datastore to a Kubernetes datastore.
+canonical_url: '/maintenance/datastore-migration'
+---
+
+## Big Picture
+
+Switch the Calico datastore from etcdv3 to Kubernetes.
+
+## Value
+
+In order to utilize all of Calico's newest features, Calico backed by
```

Comment:
To utilize all of {{site.prodname}}'s newest features, Kubernetes datastore is required. We provide a seamless way to migrate your data from an existing cluster with an etcdv3 datastore to a Kubernetes datastore. For the advantages of using a
Kubernetes datastore over an etcd datastore, see 
[Calico Datastore]({{ site.baseurl }}/getting-started/kubernetes/hardway/the-calico-datastore#using-kubernetes-as-the-datastore)
documentation.